media/js/firefox/all/all-downloads-unified.js

```diff
@@ -443,6 +443,7 @@
      * @param {Object} event object.
      */
     FirefoxDownloader.onPlatformChange = function (e) {
+        form.setAttribute('data-download-version', e.target.value);
```

Comment:
Question: Is this fixing something? The attributes should already get set in `setDownloadLink()` as far as I can see?

Reply:
I added that to the form to use as a selector to show/hide the debian text that shows up at the bottom of the page near the download link. Let me mess around and maybe we can avoid doing this.

Reply:
If this needs to stay, can we add a code comment to try and explain why it's here and what it's being used for?